### ISSUE TITLE
test "conv_bn_add" fails on CI with MIOpen 2.3 and above

### ISSUE THREAD

**Issue by kahmed10** [ROCmSoftwarePlatform/AMDMIGraphX#570]
Currently disabled for now in #551, but when testing on local machines, there are no issues.

**Comment by mvermeulen** [comment]
Also see same error with ORT unit tests with three models.

**Comment by mvermeulen** [comment]
Re-enable test and see if it still fails.

**Comment by scxiao** [comment]
Tried enabling this unit test, it still failed in CI build. Here is the error info:

```
MIOpen(HIP): Error [BaseGetSolution] result.out_pix_tile1 == 0 || result.out_pix_tile0 == 0
[2021-05-17T03:43:36.108Z] FAILED: test_conv_bn_add
[2021-05-17T03:43:36.108Z]     what(): /var/jenkins/workspace/AMDMIGraphX_add-conv_bn_add-test/src/targets/gpu/fuse_ops.cpp:131: compile: Compiling fusion plan failed
```

**Comment by mvermeulen** [comment]
Fails in CI, but not outside CI.
Also see this in other examples such as alexnet model.

@JehandadKhan any clues from the error message of what might lead to this?

**Comment by causten** [comment]
MI Open will not be making any changes to the current 4.5 so moving out of our 1.3 release

**Comment by aserio** [comment]
@causten Does this need to move to the next milestone?

**Comment by causten** [comment]
There is an internal issue against MIOpen to investigate.  MIGraphX doesn't have much control over the MIOpen schedule to debug.  I removed the target.

**Comment by pfultz2** [comment]
I think this was fixed in #1362.
